python/main.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from pipes import Template
```

Comment:
What is this import? This should be removed

Reply:
no idea what it is but i will remove it